config/bref.php

```diff
@@ -30,4 +30,16 @@
 
     'request_context' => false,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Write logs as JSON
+    |--------------------------------------------------------------------------
+    |
+    | By default, logs are written to CloudWatch as plain text.
+    | If you prefer, you can have them logged as JSON objects instead.
+    |
+    */
+
+    'logs_as_json' => false,
```

Comment:
One thing I wonder is "is it worth having an option for an option that already exists?" (`logging.channels.stderr.formatter`)

Reply:
Since Laravel (or well, Monolog probably) handles the formatting of logs, I'd say leave it there. Suggestion in the docs/readme makes sense.

Reply:
Understood, I pushed a new commit to change that 👍